package.json

```diff
@@ -32,6 +32,7 @@
   "peerDependencies": {
     "@unbrace/hooks": ">=0.0.3",
     "react": "^16.8.0",
+    "react-content-loader": "^5.0.4",
```

Comment:
Make sure to also add this to the dev deps, otherwise you're unable to run it in storybook.